src/kubectlUtils.ts

```diff
@@ -101,6 +110,50 @@ export async function getNamespaces(kubectl: Kubectl): Promise<Namespace[]> {
     });
 }
 
+export async function getDeployments(kubectl: Kubectl): Promise<Deployment[]> {
+    const currentNS = await currentNamespace(kubectl);
+
+    const shellResult = await kubectl.invokeAsync("get deployments -o json --namespace=" + currentNS);

```

Comment:
you can use some new fangled template strings here! 

ie:
```
kubectl.invokeAsync(`get deployments -o json --namespace=${currentNS}`)
```

Reply:
Done.